src/Pages/ProductDetail/RoomIntroduce/ImgArea/ImgArea.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styled from "styled-components";
+
+function ImgArea({ imgUrl }) {
+  const quterImgUrl = imgUrl.filter((el, idx) => idx !== 0);
```

Comment:
```suggestion
  const quterImgUrl = imgUrl.slice(1)
```
이건 어떨까요?

아니면 이렇게도 가능합니다
```
const [fooImg, ...restImg] = imgUrl
```

Reply:
Wow